shared/constants/swaps.js

```diff
@@ -10,6 +10,8 @@ export const SWAPS_FETCH_ORDER_CONFLICT = 'swaps-fetch-order-conflict';
 // An address that the metaswap-api recognizes as ETH, in place of the token address that ERC-20 tokens have
 const ETH_SWAPS_TOKEN_ADDRESS = '0x0000000000000000000000000000000000000000';
 
+const BNB_SWAPS_TOKEN_ADDRESS = '0x0000000000000000000000000000000000000000';
```

Comment:
Nit: Maybe instead of adding an additional constant, we could rename the old one to no longer imply that it's network-specific? It doesn't seem network-specific.

Reply:
good suggestion. Done in 2ee45891e